mlrun/frameworks/lightgbm/__init__.py

```diff
@@ -0,0 +1,53 @@
+"""
+Description:
+__init__ function of LightGBM-autologger. Will be extended and contain multiple LightGBM-specific functions.
+"""
+
+import mlrun
+from mlrun.frameworks._common.pkl_model_server import PickleModelServer
+from mlrun.frameworks.mlbase.mlrun_interface import MLBaseMLRunInterface
+
+# Temporary placeholder, LGBMModelServer may deviate from PklModelServer in upcoming versions.
+LGBMModelServer = PickleModelServer
+
+
+def apply_mlrun(
+    model,
+    context: mlrun.MLClientCtx = None,
+    X_test=None,
+    y_test=None,
+    model_name=None,
```

Comment:
Take a look at the tf_keras or pytorch code, there are typing hints for every known type. For example: model_name: str.

Remember, the better the code is written and documented, the less questions you will receive.